account-keys/src/account_keys.rs

```diff
@@ -605,4 +762,27 @@ mod account_key_tests {
         // verifier.
         verify_signature(&subaddress, &fog_authority_spki);
     }
+
+    #[test]
+    // Account Key and View Account Key derived from same keys should generate the
+    // same subaddresses
+    fn test_view_account_keys_subaddresses() {
+        let mut rng: StdRng = SeedableRng::from_seed([42u8, 32]);
+        let view_private = RistrettoPrivate::from_random(&mut rng);
+        let spend_private = RistrettoPrivate::from_random(&mut rng);
+        let account_key = AccountKey::new(&spend_private, &view_private);
+        let view_account_key = ViewAccountKey::from(&account_key);
+
+        assert_eq!(
+            account_key.default_subaddress(),
+            view_account_key.default_subaddress()
+        );
+
+        assert_eq!(
+            account_key.change_subaddress(),
+            view_account_key.change_subaddress()
+        );
+
+        assert_eq!(account_key.subaddress(500), view_account_key(500));
```

Comment:
Seems to be missing the method name.
```suggestion
        assert_eq!(account_key.subaddress(500), view_account_key.subaddress(500));
```